akd/src/storage/types.rs

```diff
@@ -38,6 +40,10 @@ pub struct Values(pub String);
 #[derive(Debug, Clone, Hash, Eq, PartialEq, Serialize, Deserialize)]
 pub struct ValueStateKey(pub String, pub u64);
 
+/// Root hash of the tree and its associated epoch
+#[derive(Debug, Clone, Hash, Eq, PartialEq, Serialize, Deserialize)]
+pub struct EpochHash<H: Hasher>(pub u64, pub H::Digest);
+
```

Comment:
Since ```EpochHash<H>``` isn't actually a "storage" type, can you move this to directory.rs? Just since the storage layer has no interaction with this type.